src/features/ITGovernance/Admin/GRBReview/DecisionRecord/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { GridContainer, Icon } from '@trussworks/react-uswds';
+import { SystemIntakeGRBReviewerFragment } from 'gql/generated/graphql';
+
+import Breadcrumbs from 'components/Breadcrumbs';
+import IconLink from 'components/IconLink';
+
+import DecisionRecordTable from './_components/DecisionRecordTable';
+
+type DecisionRecordProps = {
+  systemIntakeId: string;
+  grbReviewers: SystemIntakeGRBReviewerFragment[];
+};
+
+/**
+ * System intake GRB review decision record page
+ * */
+const DecisionRecord = ({
+  systemIntakeId,
+  grbReviewers
+}: DecisionRecordProps) => {
+  const { t } = useTranslation('grbReview');
+
+  return (
+    <GridContainer className="padding-bottom-10">
+      <Breadcrumbs
+        items={[
+          { text: t('Home'), url: '/' },
+          {
+            text: t('governanceReviewTeam:itGovernanceRequestDetails'),
+            url: `/it-governance/${systemIntakeId}/intake-request`
+          },
+          {
+            text: t('decisionRecord.breadcrumb')
+          }
+        ]}
+      />
+
+      <h1 className="margin-top-5 margin-bottom-105">
+        {t('decisionCard.heading')}
+      </h1>
+
+      <p className="font-body-lg text-light margin-top-0">
+        {t('decisionRecord.description')}
+      </p>
+
+      <IconLink
+        icon={<Icon.ArrowBack />}
+        to={`/it-governance/${systemIntakeId}/intake-request`}
+      >
+        {t('decisionRecord.returnToRequestDetails')}
+      </IconLink>
+
+      <DecisionRecordTable grbReviewers={grbReviewers} />
+    </GridContainer>
+  );
+};
```

Comment:
I would imagine this link would go back to the GRB review details (considering the GRB review tab is the only way to get to this current page), might be worthwhile to check with UX.

Reply:
Good catch! I updated the link in this button as well as the breadcrumbs.